src/main/java/com/google/codeu/servlets/MessageFeedServlet.java

```diff
@@ -0,0 +1,22 @@
+package com.google.codeu.servlets;
```

Comment:
This pull requests contains mostly datastore changes. Hence you should ideally make the servlet changes a separate pull request for modularity.

Reply:
Sure! I have a question: should the PR be separated although this branch is the "main" feature branch for public feed? Since all modular features have been merged into this one.